AGENTS.md

```
@@ -1,282 +1,18 @@
-| `clean-code` | Ao revisar ou refatorar qualquer código |
-| `web-quality-skills` | Ao revisar qualidade geral do frontend (acessibilidade, performance) |
-| `auth-implementation-patterns` | Ao implementar fluxos JWT / Google OAuth |
-
-### Skills de segurança
+Este repositório é gerenciado por uma equipe de agentes de IA em parceria com o **Orquestrador Humano (Zen Eyer)**.
 
-| Skill | Quando usar |
-|---|---|
-| `codeql-security` | Ao escrever sanitização PHP, escaping, prepared statements |
-| `backend-security-coder` | Ao implementar endpoints com inputs externos no PHP |
+## 🚪 Ponto de Entrada para IAs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)